### PR TITLE
Add PBR 3D Gen-I printer definition

### PR DESCRIPTION
**Pull request by pbr-research**
Hello Team Cura,
We are happy to introduce our new 3d printer with all 3D printing Enthausiast. We wants to add our printer profile definition i.e. PBR 3D Gen-I in ultimaker Cura slicer.

We had tested it locally on our system.
Screenshot (182)
Screenshot (183)
Screenshot (184)

### CODE REVIEW

**Comment by konskarm** [comment]
Hi @pbr-research. The printer seems to work fine in Cura, but I cannot find a webpage that has the printer's specifications. Could you provide one?

**Comment by pbr-research** [comment]
Please find our printer details on
https://www.pbrresearch.com/product/pbr-3d-gen-i/

File: resources/definitions/pbr3d_g1.def.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        },
+        "machine_width": {
+            "default_value": 180
+        },
+        "machine_height": {
+            "default_value": 200
+        },
+        "machine_depth": {
+            "default_value": 180
+        },
```
[the remark this answers is not in the document]
Comment (ksonone) [reply resources/definitions/pbr3d_g1.def.json:31]: @konskarm 
Please ensure that 180x180x200 mm is the actual buildplate dimension.

**Review by pbr-research — COMMENTED** [review]
update the profile details

**Comment by konskarm** [comment]
I see no issues anymore, so I am merging this. The printer PBR 3D Gen-I should appear in the printer list starting from Cura version 4.11.

**Review by konskarm — APPROVED** [review]

**Pull request merged**